projects/bzip2/bzip2_compress_target.c

```diff
@@ -0,0 +1,67 @@
+/*
```

Comment:
Just curious, what's the speed like on this fuzz target? 
I don't think there's a right answer to whether we should compress and then decompress. I generally think this is a good idea (I've mentioned this to you in the last CL), but if it is very slow then it might be harmful.

Reply:
It averages 1.5k execs/s on an i7 CPU @ 2.90GHz. Is this slow? I've noticed that speeds on clusterfuzz are usually slower than on my laptop possibly due to virtualized infra.

Reply:
>Is this slow?
No I would say this is pretty good.

>I've noticed that speeds on clusterfuzz are usually slower than on my laptop possibly due to virtualized infra.

Right, I notice this too, I assume the cpus/disks etc. are lower end on ClusterFuzz than they are on my desktop.